src/spec-node/upgradeCommand.ts

```diff
@@ -123,4 +146,72 @@ async function featuresUpgrade({
 	}
 	await dispose();
 	process.exit(0);
+}
+
+async function updateFeatureVersionInConfig(params: { output: Log }, config: DevContainerConfig, configPath: string, targetFeature: string, targetVersion: string) {
+	const { output } = params;
+
+	if (!config.features) {
+		// No Features in config to upgrade
+		output.write(`No Features found in '${configPath}'.`);
+		return;
+	}
+
+	if (!configPath || !(await isLocalFile(configPath))) {
+		throw new ContainerError({ description: `Error running upgrade command.  Config path '${configPath}' does not exist.` });
+	}
+
+	const configText = await readLocalFile(configPath);
+	const previousConfigText: string = configText.toString();
+	let updatedText: string = configText.toString();
+
+	// Clear Features object to make editing easier
```

Comment:
Wouldn't it be easier to only edit the affected property inside "features"? (Also: jsonc.parse drops all formatting and comments, so these would be lost from all features' options.)

Reply:
Probably :) Will give it a try

Reply:
Made a change, not sure if it's much better though.  Do you think I should do this a different way (eg: without jsonc)?

Reply:
Changed it to a simple regex in https://github.com/devcontainers/cli/pull/684/commits/c13b2c228899026829c9e6ca7384f9d9db856d15

Reply:
LGTM! I realize I took a similar approach for the 'code lenses' in the Dev Containers extension (doing string replacements).

Reply:
I've done the same thing in the past for the `template apply` function